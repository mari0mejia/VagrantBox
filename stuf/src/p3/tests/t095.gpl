int apple[3+4];
double banana[3*4];
string carrot[12/3];
