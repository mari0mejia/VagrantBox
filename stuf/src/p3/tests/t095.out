gpl.cpp::main()
  input file(tests/t095.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int apple[0] = 0
int apple[1] = 0
int apple[2] = 0
int apple[3] = 0
int apple[4] = 0
int apple[5] = 0
int apple[6] = 0
double banana[0] = 0
double banana[1] = 0
double banana[2] = 0
double banana[3] = 0
double banana[4] = 0
double banana[5] = 0
double banana[6] = 0
double banana[7] = 0
double banana[8] = 0
double banana[9] = 0
double banana[10] = 0
double banana[11] = 0
string carrot[0] = ""
string carrot[1] = ""
string carrot[2] = ""
string carrot[3] = ""
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
